docs/tracking-events.md

```diff
@@ -250,8 +254,109 @@ class Page extends Component {
 }
 ```
 
+## Track a Product Clicked event
+
+Tracking the `Product Viewed` Segment event provided HOC `trackProductViewed`.
+
+See `src/components/ProductDetail/ProductDetail.js` for the full example.
+
+```js
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import track from "lib/tracking/track";
+import trackProductClicked from "lib/tracking/trackProductClicked";
+
+// Product list item.
+@track()
+class ProductItem extends Component {
+  static propTypes = {
+    product: PropTypes.object.isRequired
+  }
+
+  // Track event on component mount,
+  // trackProductClicked, expects the prop `product` to be supplied to component
+  @trackProductClicked()
+  handleClick() {
+    // Do something on product clicked, like routing to a new page or view
+  }
+
+  render() {
+    return (
+      <button onClick={this.handleClick}>
+        {this.props.product.title}
+      </button>
+    );
+  }
+}
+```
+
 # Segment events and data mappings
 
+## Product list viewed event
+
+Data for the Segment e-commerce event [Product List Viewed](https://segment.com/docs/spec/ecommerce/v2/#product-list-viewed), will require values from a `CatalogItemProduct` and `CatalogItemVariant`.
+
+```js
+{
+  // ID of the list being viewed. Should be a tag id or home if it's the homepage
+  list_id: "",
+
+  // Name of tag or Home
+  category: "",
```

Comment:
`Product list viewed` does not seem to be getting the `category` data, it's always blank, no matter which list I'm on.

Reply:
@kieckhafer I'm getting the category when I visit a tag page

![screen shot 2018-06-12 at 11 01 32 am](https://user-images.githubusercontent.com/42217/41308158-01988c1a-6e30-11e8-835a-e8d32ab93638.png)